content/app/guides/multi-app-solution/_index.en.md

```diff
@@ -0,0 +1,64 @@
+---
+title: General approach for making a multi-app solution in Altinn
+linktitle: Multi-app solution
+description: Considerations and explanations of how to go about when creating a multi-app solution
+weight: 250
+aliases:
+
+- /app/multi-app-solution/
+
+---
+
+Before continue reading these guidelines, please consider if
+a multi-app solution is what you need to fulfill the purpose
+of your form(s).
+
+## What is a multi-app solution?
+
+A multi-app solution is a solution consisting of two or more
+cooperating apps, where typically (as per now) the "
+original" application(s) will trigger the creation of a new
+instance of the receiving application. As a part of the
+instantiation of the receiving application it is possible
+prefill the instance with specific data from the running
+instance of the original application.
+
+## Do I need a multi-app solution?
+
+A multi-app solution is in most cases not a necessary
+architectural choice for an Altinn form.
+
+### Use cases where you can consider utilize a multi-app solution:
+
```

Comment:
Not sure if I understand or agree with the points listed.

We might need to work a bit more on this section. Might want to focus more on WHY choose it, what introduces a need rather than "This is something you must accept if you choose this type of solution"

- My forms will be answered by users that does not have Altinn
  - Two apps don't vhange the fact that the end user would need to interact with an app at some point? 
  - We also have support for anonymous users and self identified users, or integrations with 3rd paqrty systems, e.g. Feide
-  It is okay that my receiving forms must be deleted in order to end the lifecycle of the form.
   -  not sure what is meant here? Does a two app solution require auto deletion on one or more apps? or does this address complete confirm of recipient app?

Reply:
I found this section difficult to write, so I would love some "sparring" on the topic in person later on, if you have time! ☺️